element_lab/lab.py

```diff
@@ -117,26 +147,26 @@ class ProtocolType(dj.Lookup):
         protocol_type ( varchar(32) ): Protocol types (e.g., IACUC, IRB, etc.).
     """
 
-    definition = """
-    protocol_type           : varchar(32)
+    definition = """# Type of protocol or issuing agency
+    protocol_type           : varchar(32)  # Protocol types (e.g., IACUC, IRB, etc.)
     """
 
 
 @schema
 class Protocol(dj.Lookup):
-    """Protocol specifics (e.g., protocol number and title).
+    """Protocol approved by some institutions like IACUC, IRB, or experimental protocol.
 
     Attributes:
-        protocol ( varchar(16) ): Protocol identifier.
+        protocol ( varchar(36) ): Protocol identifier.
         ProtocolType (foreign key): ProtocolType key.
         protocol_description( varchar(255) ): Optional. Description of the protocol.
```

Comment:
```suggestion
        protocol_description( varchar(255), optional ): Description of the protocol.
```

Reply:
I believe we have settled on the above method of describing nullable attributes in the docstrings.